.github/workflows/releasability.yml

```diff
@@ -0,0 +1,27 @@
+name: Releasability status
+'on':
+  check_suite:
+    types:
+      - completed
+jobs:
+  update_releasability_status:
+    runs-on: ubuntu-latest
+    name: Releasability status
+    permissions:
+      id-token: write
+      statuses: write
+      contents: read
+    if: >-
+      (contains(fromJSON('["main", "master"]'),
+      github.event.check_suite.head_branch) ||
+      startsWith(github.event.check_suite.head_branch, 'dogfood-') ||
+      startsWith(github.event.check_suite.head_branch, 'branch-')) &&
+      github.event.check_suite.conclusion == 'success' &&
+      github.event.check_suite.app.slug == 'cirrus-ci'
+    steps:
+      - uses: >-
+          SonarSource/gh-action_releasability/releasability-status@v2
+        with:
+          optional_checks: "Jira,CheckDependencies"
```

Comment:
Why is `CheckDependencies` optional?

Reply:
Good question! During the development cycle, we often depend on artifacts that are not yet released (mostly sonarlint-core).

Oftentimes, the release of `sonarlint-core` happens shortly before the release of `sonarlint-language-server`